### PR TITLE
clarify uninstall

### PR DESCRIPTION
**Pull request by robinbobbitt**
"Change to your project namespace " is unclear which project you're talking about... Thinking the suggested change should be more clear to users.

### CODE REVIEW

File: uninstall.adoc [hunk 1]
```diff
@@ -16,22 +16,16 @@ Note: When a cluster's authentication is no longer managed by identity configura
...
-❯ oc get csv
-NAME                         DISPLAY                                                           VERSION   REPLACES   PHASE
-idp-mgmt-operator.v1.0.0   identity configuration management for Kubernetes for Kubernetes   1.0.0                Succeeded
+❯ oc get csv -A
+NAMESPACE            NAME                         DISPLAY                                                           VERSION   REPLACES   PHASE
+idp-mgmt-config      idp-mgmt-operator.v1.0.0     identity configuration management for Kubernetes for Kubernetes   1.0.0                Succeeded
 
-❯ oc delete clusterserviceversion idp-mgmt-operator.v1.0.0
-❯ oc delete sub idp-mgmt-operator
+❯ oc delete clusterserviceversion idp-mgmt-operator.v1.0.0 -n idp-mgmt-config
```
Comment (chrisahl) [inline uninstall.adoc:27]: We are not restricting the namespace to `idp-mgmt-config` correct?

File: uninstall.adoc [hunk 2]
```diff
@@ -16,22 +16,16 @@ Note: When a cluster's authentication is no longer managed by identity configura
 [#removing-a-identityconfiguration-instance-by-using-commands]
 == Removing resources by using commands
 
-. If you have not already. ensure that your OpenShift Container Platform CLI is configured to run `oc` commands. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the OpenShift CLI] in the OpenShift Container Platform documentation for more information about how to configure the `oc` commands. 
+. If you have not already, ensure that your OpenShift Container Platform CLI is configured to run `oc` commands. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the OpenShift CLI] in the OpenShift Container Platform documentation for more information about how to configure the `oc` commands. 
 
-. Change to your project namespace by entering the following command. Replace _namespace_ with the name of your project namespace:
-+
-----
-oc project <namespace>
```
Comment (chrisahl) [inline uninstall.adoc:24]: Guess using project depends if we want to use some OCP terminology vs kube terminology.

Comment (robinbobbitt) [reply uninstall.adoc:24]: I wasn't concerned with the use of the term "project", more that we won't know which project we're referring to.. in this case the one where they installed the operator. I'm fine with an alternate fix if you want to keep `oc project` in the mix!

File: uninstall.adoc (hunk 1 above)
Comment (robinbobbitt) [reply uninstall.adoc:27]: clarification added in second commit

**Review by chrisahl — APPROVED** [review]

**Pull request merged**